.pre-commit-config.yaml

```diff
@@ -16,3 +16,10 @@ repos:
       - id: codespell
         additional_dependencies:
           - tomli
+  - repo: local
+    hooks:
+    - id: dipy-commit-msg
+      name: 'dipy-commit-msg'
+      entry: 'tools/hooks/commit-msg.py'
+      language: python
+      stages: [commit-msg]
```

Comment:
`[commit-msg]` not sure if this tell pre-commit to check only locally at the commit moment. At the CI's level, the commit is already done and push so it would make sense to me to not show up here. 

Soorrry, this decrease in priority but I will look at it at some point.

Reply:
Thanks for the review @skoudoro. Makes sense. Will remove, force-push and see if the pre-commit job reports the status.

Reply:
The issue may be that the commit message file does not actually exist in the CI runner.

Reply:
Ran out of ideas:
https://stackoverflow.com/questions/79704142/make-pre-commit-work-to-check-the-commit-message-on-github